dags/transportation/dockless/dockless_elt.py

```diff
@@ -217,6 +217,7 @@ def load_to_s3(**kwargs):
 status_changes = """
 
 CREATE TABLE IF NOT EXISTS status_changes (
+    id SERIAL PRIMARY KEY 
```

Comment:
This is unfortunate. I know you said there was some history here -- any forthcoming ways to more reliably assign a status change id?

Reply:
yeah, unfortunately there isn't a status_change_id in the spec. 

the alternate approach is joining on the constraints from the CREATE_TABLE statements, but those haven't been entirely reproduceable